### ISSUE TITLE
2.0 template ?

### ISSUE THREAD

**Issue by stefanpenner** [ember-cli/ember-twiddle#150]

**Comment by Gaurav0** [comment]
We need to wait until 2.0 is on the CDN.

**Comment by Gaurav0** [comment]
We've decided to table this for a few weeks and reevaluate.

In the meantime, if anyone wants to use Ember 2.0, here is the template you need:

    {
       "version": "0.4.8",
       "dependencies": {
        "jquery": "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.js",
        "ember": "https://cdnjs.cloudflare.com/ajax/libs/ember.js/2.0.0/ember.js",
        "ember-data": "https://cdnjs.cloudflare.com/ajax/libs/ember-data.js/2.0.0-beta.1/ember-data.js",
        "ember-template-compiler": "https://cdnjs.cloudflare.com/ajax/libs/ember.js/2.0.0/ember-template-compiler.js"
      }
    }

**Comment by stefanpenner** [comment]
> In the meantime, if anyone wants to use Ember 2.0, here is the twiddle.json you need:

i wonder if down the road we want a list of curated twiddle.json's

**Comment by greyhwndz** [comment]
> i wonder if down the road we want a list of curated twiddle.json's – could be handy
@stefanpenner: I think that would be great! for 2.0.1 would the following be okay?
```
{
   "version": "0.4.8",
   "dependencies": {
    "jquery": "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.js",
    "ember": "https://cdnjs.cloudflare.com/ajax/libs/ember.js/2.0.1/ember.js",
    "ember-data": "https://cdnjs.cloudflare.com/ajax/libs/ember-data.js/2.0.0-beta.2/ember-data.js",
    "ember-template-compiler": "https://cdnjs.cloudflare.com/ajax/libs/ember.js/2.0.0/ember-template-compiler.js"
  }
}
```
Would the ember-template-compiler version be okay for 2.0.1?

**Comment by Gaurav0** [comment]
@greyhwndz The version of the ember-template-compiler should always match the version of ember.

**Comment by greyhwndz** [comment]
@Gaurav0 Thanks for the info :+1:

**Comment by alexspeller** [comment]
This is resolved by #226
